src/main/scala/com/databricks/spark/avro/package.scala

```diff
@@ -44,4 +45,12 @@ package object avro {
   implicit class AvroDataFrameReader(reader: DataFrameReader) {
     def avro: String => DataFrame = reader.format("com.databricks.spark.avro").load
   }
+  
+    /**
+   * Adds a method, `avro`, to DataFrameReader that allows you to read avro files using
+   * the DataFileReade
+   */
+  implicit class AvroDataFrame(dataFrame: DataFrame) {
+    def addAvroAliasColumns(): DataFrame = SchemaConverters.dataFrameWithAliasColumn(dataFrame)
+  }
```

Comment:
Can we get rid of this entirely?  I'd prefer to only have to support a single unified way to do this.